source/_integrations/zha.markdown

```diff
@@ -49,9 +49,9 @@ Some other Zigbee coordinator hardware may not support a firmware that is capabl
 ### Known working Zigbee radio modules
 
 - dresden elektronik deCONZ based Zigbee radios (via the [zigpy-deconz](https://github.com/zigpy/zigpy-deconz) library for zigpy)
-  - [ConBee II (a.k.a. ConBee 2) USB adapter from dresden elektronik](https://phoscon.de/conbee2)
+  - [ConBee II (a.k.a. ConBee 2) USB adapter from dresden elektronik](https://phoscon.de/conbee2) (Note! Not a must but recommend [upgrade the deCONZ application firmware for ConBee II](https://github.com/dresden-elektronik/deconz-rest-plugin/wiki/Update-deCONZ-manually))
   - [ConBee USB adapter from dresden elektronik](https://phoscon.de/conbee)
-  - [RaspBee II (a.k.a. RaspBee 2) Raspberry Pi Shield from dresden elektronik](https://www.dresden-elektronik.com/product/raspbee-II.html)
+  - [RaspBee II (a.k.a. RaspBee 2) Raspberry Pi Shield from dresden elektronik](https://www.dresden-elektronik.com/product/raspbee-II.html)  (Note! Not a must but recommend [upgrade the deCONZ application firmware for ConBee II](https://github.com/dresden-elektronik/deconz-rest-plugin/wiki/Update-deCONZ-manually))
   - [RaspBee Raspberry Pi Shield from dresden elektronik](https://phoscon.de/raspbee)
 - Silicon Labs EmberZNet based radios using the EZSP protocol (via the [bellows](https://github.com/zigpy/bellows) library for zigpy)
```

Comment:
I'm really sorry, but the list of supported radio modules has become unreadable list. We can just keep adding notes and links to each item.

Reply:
Maybe just a separate "Firmware" section further down with all those links? Or could we have subpages to integrations?